packages/sdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@microsoft/teamsfx",
-  "version": "0.4.0-alpha.3",
+  "version": "0.3.0-alpha.0",
```

Comment:
Do we need to rollback package-lock.json?

Reply:
WIP